### ISSUE TITLE
where is the generator ?

### ISSUE THREAD

**Issue by cmchao** [michaeljclark/riscv-meta#3]
I get the repo link from qemu saying the most of disas part comes from here.   But I can't find an program to read meta and produce  disassembler part.

**Comment by Nlossae** [comment]
correct, you need the `rv-meta` tool from [rv8](https://github.com/rv8-io/rv8/)
